addons/website_livechat/static/src/web/channel_member_model_patch.js

```diff
@@ -4,8 +4,8 @@ import { patch } from "@web/core/utils/patch";
 /** @type {import("models").ChannelMember} */
 const channelMemberPatch = {
     getLangName() {
-        if (this.persona.is_public && this.thread.visitor?.langName) {
-            return this.thread.visitor.langName;
+        if (this.persona.is_public && this.thread.livechat_visitor_id?.lang_id.name) {
+            return this.thread.livechat_visitor_id.lang_id.name;
         }
```

Comment:
It seems that this condition is never met. `is_public` comes with the partner fields, and if there is a partner as a visitor, it's not public. `is_public` field was introduced in the partner data since #98923 and then used to show the lang of the visitor in #128242, but at that time it was possible to have a website visitor as a public user. Since #129770 the visitors are guests, and we don't send and use `is_public` for guests anymore.